.github/.hintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "connector": {
+    "name": "local",
+    "options": {
+      "pattern": ["**", "!.git/**", "!node_modules/**"]
+    }
+  },
+  "extends": ["development"],
+  "formatters": ["stylish"],
+  "hints": [
+    "button-type",
+    "disown-opener",
+    "html-checker",
+    "meta-charset-utf-8",
+    "meta-viewport",
+    "no-inline-styles:error"
+  ]
+}
```

Comment:
- [ Optional ] To make your code-base clean Kindly, Remove ".hintrc",".stylelintrc.json","package.json" and "package-lock.json" files from "github" folder they already exist in your root directory .

Reply:
Okay Thank you @Abenezer-Tilahun😊